src/assets/_variables.scss

```diff
@@ -1,11 +1,11 @@
-$primary-white: #ffffff;
-$primary-black: #000000;
+$primary-white: #fff;
```

Comment:
Good job all in all... I noticed you didn't go back to the underscored naming we had the very first time. Is it because we don't want to use it again or sass-lint cannot handle it?

Reply:
Yes that is one part that they do not have out of the box
you either have to use hyphen all through, or underscore of go with the hyphenated-BEM`(block__element--modifier)`